packages/devtools/lib/src/inspector/diagnostics_node.dart

```diff
@@ -149,7 +149,7 @@ class RemoteDiagnosticsNode extends DiagnosticableTree {
 
   /// Hint for how the node should be displayed.
   DiagnosticsTreeStyle get style {
-    return _styleOverride ??
+    return _styleOverride ??=
```

Comment:
change this method from _styleOverride to 
_style 
now that it is more than just an override.